fiboa_cli/datasets/commons/admin.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, *args, **kwargs):
         if self.admin_country_code is None:
             # Taking country code from ID
             self.admin_country_code = self.id.split("_")[0].upper()
-        assert re.match("[A-Z]{2}", self.admin_country_code), (
+        assert re.match("[A-Z]{2}$", self.admin_country_code), (
```

Comment:
```suggestion
        assert re.match("^[A-Z]{2}$", self.admin_country_code), (
```

See also: https://fiboa.org/administrative-division-extension/v0.1.0/schema.yaml
I'm wondering, whether we need to check it here if the schema also validates it?